components/instructions/tools.tsx

```diff
@@ -30,11 +30,17 @@ import { NAME_SERVICE_INSTRUCTIONS } from './programs/nameService'
 import { TOKEN_AUCTION_INSTRUCTIONS } from './programs/tokenAuction'
 import { VALIDATORDAO_INSTRUCTIONS } from './programs/validatordao'
 
+export const V3_DEFAULT_GOVERNANCE_PROGRAM_ID =
+  '7e75Nwsz8i5i4NiDa43CNzKJ4AeQGyRimha46VKTM1Ls'
+
+export const V2_DEFAULT_GOVERNANCE_PROGRAM_ID =
+  'GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw'
+
 /**
  * Default governance program id instance
  */
-export const DEFAULT_GOVERNANCE_PROGRAM_ID =
-  'GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw'
+export const DEFAULT_GOVERNANCE_PROGRAM_ID = V2_DEFAULT_GOVERNANCE_PROGRAM_ID
```

Comment:
This makes it easy to flip the switch to v3; just change this value. But we shouldn't forget the v2 default.